wagtail/utils/timestamps.py

```diff
@@ -0,0 +1,11 @@
+from django.utils import formats, timezone
+
+
+def render_timestamp(timestamp):
```

Comment:
Is it ok to introduce a new module to utils?
Further is the naming ok?

Reply:
Thank you for the work. Great catch!

We might also use:

```
from django.template.defaultfilters import date
date(value, "DATETIME_FORMAT")
```

See also my comment on the issue https://github.com/wagtail/wagtail/issues/9581#issuecomment-1302309601

With this simple syntax, I'm not sure we need a utility method.

Just 2ct. Love the effort!

Reply:
Good point, it would have been nice that way :+1: 
Unfortunately that doesn't work :smile:  https://github.com/wagtail/wagtail/issues/9581#issuecomment-1302315202

Reply:
We already have this exact implementation here:

https://github.com/wagtail/wagtail/blob/9537547b46c4f07938779103c7c42ba424fbe9a6/wagtail/test/utils/timestamps.py#L22-L29

I'm in favour of having this outside of the test module though, so I suppose we can remove that one and change imports to use this one.

Reply:
@laymonage good point, it looks like I've copied it, but also adapted it to use the l18n dependent format instead of the default format (from the settings), which is better for user-faced timestamps. Maybe that's the reason, why I haven't replaced the version from the tests, but I don't know why I did not mention that above :thinking: 

Also I now think there should not be a problem :sweat_smile: 
I will try to replace the test utility with the above one, when I'm rebasing the PR.

Reply:
Speaking about rebasing, #9628 is a tiny fix, which could be reviewed/merged, but it intersects a bit with this PR (tests), so having it merged would help me rebasing this. Sorry for using this thread, but could you do that @laymonage ?

Reply:
I think this thread has been resolved.

Reply:
You're right, thx!